contracts/adapters/berezka/BerezkaTokenAdapterGovernance.sol

```diff
@@ -26,11 +40,17 @@ import { Ownable } from "../../Ownable.sol";
  */
 contract BerezkaTokenAdapterGovernance is Ownable() {
 
+    AdapterRegistry immutable adapterRegistry = AdapterRegistry(0x06FE76B2f432fdfEcAEf1a7d4f6C3d41B5861672);
```

Comment:
you can use `constant` as you do not set it in the constructor -- no need to use `immutable`